index.html

```diff
@@ -0,0 +1,355 @@
+              </li>
+            </ul>
+            <button type="button" class="btn clr-p-500">
+              Get my resume
+            </button>
+          </div>
+          <ul class="my__views">
+            <li class="my_languages">
+              <div class="my_languages_favourite">
+                <h3 class="my_languages_favourite__title clr-n-900">
+                  Languages
+                </h3>
+                <img class="my_languages_favourite__img" src="images/arrow.png" alt="arrow icon" />
+              </div>
+              <ul class="my_languages__bottom">
+                <li class="list">
+                  <img src="images/js.png" alt="javascript icon" />
+                  <p class="my_languages__bottom__name clr-n-700">
+                    JavaScript
+                  </p>
+                </li>
+                <li class="list">
+                  <img src="images/html.png" alt="html icon" />
+                  <p class="my_languages__bottom__name clr-n-700">
+                    HTML
+                  </p>
+                </li>
+                <li class="list">
+                  <img src="images/css.png" alt="css icon" />
+                  <p class="my_languages__bottom__name clr-n-700">
+                    CSS
+                  </p>
+                </li>
+              </ul>
+            </li>
+            <li class="my__technologies">
+              <div class="my__technologies__top">
+                <h3 class="my__technologies__top__title clr-n-900">
+                  Frameworks
+                </h3>
+                <img class="my__technologies__top__img" src="images/arrow.png" alt="arrow icon" />
+              </div>
+            </li>
+            <li class="my__skills">
+              <div class="my__skills__top">
+                <h3 class="my__skills__top__title clr-n-900">
+                  Skills
+                </h3>
+                <img class="my__skills__top__img" src="images/arrow.png" alt="arrow icon" />
+              </div>
+            </li>
+          </ul>
+        </section>
+        <section class="form-section" id="reach_me">
+            <div class="div-wrap">
+             <h2 class="form_t">Contact me</h2>
+             <p class="form-paragraph">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+             <form class="reach_me" action="https://formspree.io/f/mbjeobpq" method="post">
+               <input type="text" name="name" placeholder="Oscar Loremipsum" maxlength="30" required>
+               <input type="email" name="email" placeholder="oscar@gamil.com" required>
+               <textarea name="messages" rows="10" maxlength="500" required placeholder="Enter text here"></textarea>
+               <button type="submit" name="submit">Get in touch</button>
+               <span></span>
+             </form>
```

Comment:
<img width="383" alt="1" src="https://user-images.githubusercontent.com/77843050/221155302-505cd993-4a39-49f1-b40c-cc1f3a4d9163.png">

- [ ] Nice work @oscar-byte15, you have respected the requirement but the design is broken I think, please fix the look in the mobile version and resubmit your work, take an example from this given template [here](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=qozWkJfErHpIJG0Z-0)

ORIGINAL DESIGN

<img width="362" alt="2" src="https://user-images.githubusercontent.com/77843050/221155220-91d54ba8-b823-48bf-b5d8-8e67bdfb993f.png">

Reply:
Hi, Joskalenda, it would be so nice that you put the size of that view, because I have used the same width of the template for mobile version of Figma. So, please, I am going to google it and use another width for mobile version. Finally, that view you show is not broken, only has a width different!

Reply:
@joskalenda